src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -6751,7 +6752,7 @@ namespace VariableSpeedCoils {
 
 	//     NOTICE
 
-	//     Copyright � 1996-2014 The Board of Trustees of the University of Illinois
+	//     Copyright � 1996-2014 The Board of Trustees of the University of Illinois
```

Comment:
@nmerket As soon as #5035 merges in, pull develop into this branch and resolve the conflict in the copyright line by just choosing the new plain text version and we'll never have to worry about these again.